rdt-app/src/main/java/io/ona/rdt_app/application/RDTApplication.java

```diff
@@ -66,6 +63,8 @@ public void onCreate() {
         Fabric.with(this, new Crashlytics.Builder().core(new CrashlyticsCore.Builder().disabled(BuildConfig.DEBUG).build()).build());
 
         JobManager.create(this).addJobCreator(new RDTJobCreator());
+
+        getContext().allSharedPreferences().savePreference(IS_IMG_SYNC_ENABLED, String.valueOf(true));
```

Comment:
after one disables image upload, is that preference supposed to be remembered until they change again or after app is restarted

Reply:
Until they change.

Reply:
Then remove this line, because this will be enabling the setting every time the app is restarted

Reply:
Made a change to enable it by default if not already set.